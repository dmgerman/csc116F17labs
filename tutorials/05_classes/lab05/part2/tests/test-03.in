22:30
1:35