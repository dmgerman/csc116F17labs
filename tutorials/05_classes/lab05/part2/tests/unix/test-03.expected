Enter a time: 
You entered "22:30"
c1.get_time_string(true) returns 22:30
c1.get_time_string(false) returns 10:30pm
Enter another time: 
You entered "1:35"
Advancing c1 by 1 hours and 35 minutes.
After advancing, the time is 00:05
